web/internal/constants.bzl

```diff
@@ -23,6 +23,7 @@ DEFAULT_WEB_TEST_SUITE_TAGS = {
     ],
     "chromium-local": [
         "native",
+        "chromium-local-macos"
```

Comment:
Why do we need to add extra tags? There's also nothing macos specific about this